<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroWidget</class>
 <widget class="QWidget" name="RegistroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Registro de Cliente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="documentoLineEdit">
     <property name="placeholderText">
      <string>Documento</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nombreLineEdit">
     <property name="placeholderText">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="telefonoLineEdit">
     <property name="placeholderText">
      <string>Tel√©fono</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="correoLineEdit">
     <property name="placeholderText">
      <string>Correo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_registrar">
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
